src/commands/generate/types.ts

```diff
@@ -110,4 +142,45 @@ export default class GenerateTypes extends Base {
         }
         return variable.type.toLocaleLowerCase()
     }
+
+    private async getVariableInfoComment(variable: Variable, orgMembers: OrganizationMember[]): Promise<string> {
+        const flags = await this.parse(GenerateTypes)
+        const { 'include-descriptions': includeDescriptions, 'inline-comments': inlineComments } = flags.flags
```

Comment:
you're accessing `flags.flags` here instead of just doing `const { flags }` in the line above (I only mention this because this is how you're accessing flags in the `getTypeDefinitionLine function` so its inconsistent)